### DISCUSSION TITLE
Un-joining

### DISCUSSION THREAD

**Discussion by Flor-e** [alkem-io/alkemio#848] [Q&A] [answered]
Is there a way to un-join a hub/challenge/opportunity? Can't seem to find in anywhere.

**Comment by techsmyth** [comment, marked as answer]
Hi Flori,

This is also a gap that we need to fill, quite high up but somehow has not made it yet to the top:
https://github.com/alkem-io/alkemio/issues/817
Related is the user-organization relationship.

It is possible to do it via graphql, which I know you can do :) But that is absolutely not a general solution!

Cheers, Neil

**Reply by Flor-e** [reply]
There is a thick layer of dust on those graphql skills... I'll wait till it's released as common people functionality ;-)
